### ISSUE TITLE
 total_loss = tf.check_numerics(total_loss, 'LossTensor is inf or nan.')

### ISSUE THREAD

**Issue by 10183308** [cftang0827/pedestrian_detection_ssdlite#3]
hello ,
I meet the below problem: 

total_loss = tf.check_numerics(total_loss, 'LossTensor is inf or nan.')

the loss is nan.

could you help me?

**Comment by cftang0827** [comment]
Could you please specify which part of code? Thanks.
